torch/fx/experimental/schema_type_annotation.py

```diff
@@ -33,6 +33,24 @@ def __init__(self, module : torch.nn.Module, annotate_functionals : bool = True,
         self.annotate_modules = annotate_modules
         self.annotate_get_attrs = annotate_get_attrs
 
+        self._cur_node = None
+
+    def run_node(self, n):
+        try:
+            self._cur_node = n
+            return super().run_node(n)
+        finally:
+            self._cur_node = None
```

Comment:
this seems a bit messy, to it is more related to the abstraction for how transforms are done and not this PR in particular.

Reply:
Is there a reason this strives to preserve type for param nodes but nothing else?

Reply:
Types were set for everything else in a previous PR, it was just missing for `placeholders`. This adds the missing annotations. Though this change is actually no longer needed now that we don't rely on types, let me just remove it